### ISSUE TITLE
Change data structure

### ISSUE THREAD

**Issue by abelsiqueira** [cffbots/filtering#42]
Possibly not for this sprint.

Changing the data structure from an array of URLs to something more complicated that kept additional information about the repos would be useful in a few places:

- #18 
- #23 
- #36

**Comment by jspaaks** [comment]
I agree, see https://github.com/cffbots/filtering/issues/15#issuecomment-1022967220
